govuk-connect

```diff
@@ -0,0 +1,1060 @@
+#     -p, --port-forward SERVICE       Connect to a remote port
+#     -v, --verbose                    Enable more detailed logging
+#     -h, --help                       Prints usage information and examples
+#
+# CONNECTION TYPES
+#   app-console
+#     Launch a console for an application.  For example, a rails console when connecting to a Rails application.
+#   app-dbconsole
+#     Launch a console for the database for an application.
+#   rabbitmq
+#     Setup port forwarding to the RabbitMQ admin interface.
+#   sidekiq-monitoring
+#     Setup port forwarding to the Sidekiq Monitoring application.
+#   ssh
+#     Connect to a machine through SSH.
+#
+# MACHINE TARGET
+#   The ssh, rabbitmq and sidekiq-monitoring connection types target
+#   machines.
+#
+#   The machine can be specified by name, for example:
+#
+#     govuk connect ssh -e integration backend
+#
+#   If the hosting provider is ambiguous, you'll need to specify it prior
+#   to the name, for example:
+#
+#     govuk connect ssh -e staging aws/backend
+#
+#   If you want to connect to a specific machine, you can specify a number
+#   after the name, for example:
+#
+#     govuk connect ssh -e integration backend:2
+#
+# APPLICATION TARGET
+#   The app-console and app-dbconsole connection types target
+#   applications.
+#
+#   The application is specified by name, for example:
+#
+#     govuk connect app-console -e integration publishing-api
+#
+#   If the node class is ambiguous, you'll need to specify it prior to
+#   the name, for example:
+#
+#     govuk connect app-console -e integration whitehall_backend/whitehall
+#
+#   If you want to connect to a specific machine, you can specify a
+#   number after the name, for example:
+#
+#     govuk connect app-console -e integration publishing-api:2
+#
+# EXAMPLES
+#   govuk connect ssh --environment integration backend
+#
+#   govuk connect app-console --environment staging publishing-api
+#
+#   govuk connect app-dbconsole -e integration whitehall_backend/whitehall
+#
+#   govuk connect rabbitmq -e staging aws/rabbitmq
+#
+#   govuk connect sidekiq-monitoring -e integration
+
+
+require 'uri'
+require 'yaml'
+require 'open3'
+require 'socket'
+require 'timeout'
+require 'optparse'
+
+def bold(string)
+  "\e[1m#{string}\e[0m"
+end
+
+USAGE_BANNER = 'Usage: govuk connect TYPE TARGET [options]'
+
+EXAMPLES = <<EXAMPLES
+  govuk connect ssh --environment integration backend
+
+  govuk connect app-console --environment staging publishing-api
+
+  govuk connect app-dbconsole -e integration whitehall_backend/whitehall
+
+  govuk connect rabbitmq -e staging aws/rabbitmq
+
+  govuk connect sidekiq-monitoring -e integration
+EXAMPLES
+
+MACHINE_TARGET_DESCRIPTION  = <<DOCS
+  The ssh, rabbitmq and sidekiq-monitoring connection types target
+  machines.
+
+  The machine can be specified by name, for example:
+
+    govuk connect ssh -e integration #{bold('backend')}
```

Comment:
```suggestion
    govuk-connect ssh -e integration #{bold('backend')}
```

These all need updating.

Reply:
probably should update the same thing in the examples, and comments above as well